src/core/Chip/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Icon from '../Icon'
+import s from './styles'
+
+// Only allow lower case letters, optionally seperated by underscore
+const mdIconNamePattern = /^[a-z]+_*[a-z]+$/
+
+class Chip extends React.PureComponent {
+    onRemove = event => {
+        event.stopPropagation()
+        this.props.onRemove()
+    }
+
+    renderIcon(icon) {
```

Comment:
use `type` property Icon or Image, default to `icon` rather than regex logic to simplify code